packages/office-ui-fabric-react/src/components/SpinButton/docs/new/SpinButtonOverview.md

```diff
@@ -0,0 +1,5 @@
+A spin button (code name: `SpinButton`) allows someone to incrementally adjust a value in small steps. It's mainly used for numeric values, but other values are supported too.
```

Comment:
what other values?

Reply:
I'm guessing this refers to the ability to put units (cm) or things like that inside the text field. I agree that we should remove it though...we don't want to encourage that.